### PR TITLE
Refactored footer and changed logo on navbar in dark mode

### PR DESCRIPTION
**Pull request by FireQueen-3010**
## Proposed changes
1) Refactored footer on dark mode

image
a) Changed color of headings
b) Changed hover color on text.
c) Changed border color and background of social icons.
d) Changed color of dashes and box-shadow of footer.

2) Changed color of icon-hover on navbar in dark mode.

https://user-images.githubusercontent.com/59244300/113399090-44f5e600-93bd-11eb-8e0f-94624d0c3867.mp4

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
@Kajol-Kumari, Please review :)

File: frontend/src/components/Navbar/navbar.module.scss [hunk 1]
```diff
@@ -53,7 +53,8 @@
 }
 
 .navbar-logo-dark img:hover {
-  filter: hue-rotate(-15deg) contrast(180%) brightness(70%);
+  // filter: hue-rotate(800deg) contrast(180%) brightness(70%);
```
Comment (Kajol-Kumari) [inline frontend/src/components/Navbar/navbar.module.scss:56]: let's remove the commented line

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
- The onhover effect of the nvbar icon(can we please keep it same as we have in light mode? as I feel that's looking better)
- The onhover effect of footer texts doesn't seem to go well. Please try to make the on-hover color same as the footer's heading color. I believe that would look better.

File: frontend/src/components/Navbar/navbar.module.scss (hunk 1 above)
Comment (FireQueen-3010) [reply frontend/src/components/Navbar/navbar.module.scss]: Done with the requested changes @Kajol-Kumari , please review :)

**Comment by FireQueen-3010** [comment]
Done with the requested changes @Kajol-Kumari , please review :)

**Review by Kajol-Kumari — APPROVED** [review]
@FireQueen-3010 Thanks for incorporating the requested changes.
LGTM :+1:

**Pull request merged**